src/Form/CommitteeFeedMessageType.php

```diff
@@ -30,7 +30,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'purify_html' => true,
             ])
             ->add('published', CheckboxType::class, [
-                'label' => 'Rendre ce message visible sur la page du comité',
+                'label' => 'Publier sur la page du comité',
+                'attr' => ['class' => 'form__checkbox form__checkbox--large'],
+                'required' => false,
+            ])
+            ->add('send_notification', CheckboxType::class, [
```

Comment:
have you tested the checked value of this field? 
I'm wondering for a mapping of this field `send_notification` and the class property `sendNotification`...